tests/integration/run_tests.sh

```
@@ -0,0 +1,63 @@
+
+    if [[ -n "$TAG_FILTER" && "$test_name" != test_${TAG_FILTER}_* ]]; then
+        continue
+    fi
+
+    if [[ -n "$EXCLUDE_TAG" && "$test_name" == test_${EXCLUDE_TAG}_* ]]; then
+        continue
+    fi
+
+    rc=0
+    (
+        source "${SCRIPT_DIR}/lib/helpers.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./lib/helpers.sh was not specified as input (see shellcheck -x).

(SC1091)